website/docs/r/worker_script.html.markdown

```diff
@@ -38,30 +24,15 @@ resource "cloudflare_worker_script" "my_script" {
 
 The following arguments are supported:
 
-* `zone_id` - (Required for single-script accounts) The zone ID for the script.
-* `name` - (Required for multi-script accounts) The name for the script.
+* `name` - (Required) The name for the script.
 * `content` - (Required) The script content.
 
 ## Import
 
-### single-script
-
-To import a script from a single-script account, use an ID like `zone:ae36f999674d196762efcc5abb06b345`
-
-```
-$ terraform import cloudflare_worker_script.default zone:ae36f999674d196762efcc5abb06b345
-```
-
-where:
-
-* `ae36f999674d196762efcc5abb06b345` - the zone ID
-
-### multi-script
-
-To import a script from a multi-script account, use an id like `name:script_name`
+To import a script, use a script name, e.g. `script_name`
 
 ```
-$ terraform import cloudflare_worker_script.default name:script_name
+$ terraform import cloudflare_worker_script.default script_name
```

Comment:
I usually shy away from using user provided names for things as they can change or be reused whereas the IDs that Cloudflare generate do not. Not a big deal but something worth raising I think to save us some hassle later.

Reply:
Workers API has made a decision that script name is actually an ID too.
e.g.
```
    {
      "id": "example-script",
      "etag": "830924823da98de8429c9e9596ee79525aa2914fe32a19ca7e7aef77bfee2166",
      "modified_on": "2019-09-23T22:59:55.617676Z",
      "created_on": "2019-09-23T22:31:26.487806Z"
    },
```
becomes on API:
https://api.cloudflare.com/client/v4/accounts/cf0df04d99b840a890163aa457d8ee3d/workers/scripts/example-script

Reply:
:cool: this is great to know!